django_multitenant/backends/postgresql/base.py

```diff
@@ -1,144 +1,21 @@
-                model, [new_field.column], foreign_key=True
-            ) + self._constraint_names(
-                model,
-                [new_field.column, get_tenant_column(from_model)],
-                foreign_key=True,
-            )
-            if not fk_names:
-                self.execute(
-                    self._create_fk_sql(
-                        model, new_field, "_fk_%(to_table)s_%(to_column)s"
-                    )
-                )
-
-    # Override
-    def _create_fk_sql(self, model, field, suffix):
-        """
-        This method overrides the additions foreign key constraint sql and adds the tenant column to the constraint
-        """
-        if isinstance(field, TenantForeignKey):
-            try:
-                # test if both models exists
-                # This case happens when we are running from scratch migrations and one model was removed from code
-                # In the previous migrations we would still be creating the foreign key
-                from_model = get_model_by_db_table(model._meta.db_table)
-                to_model = get_model_by_db_table(
-                    field.target_field.model._meta.db_table
-                )
-            except ValueError:
-                return None
-
-            from_columns = field.column, get_tenant_column(from_model)
-            to_columns = field.target_field.column, get_tenant_column(to_model)
-            suffix = suffix % {
-                "to_table": field.target_field.model._meta.db_table,
-                "to_column": "_".join(to_columns),
-            }
-
-            return self.sql_create_fk % {
-                "table": self.quote_name(model._meta.db_table),
-                "name": self.quote_name(
-                    self._create_index_name(
-                        model._meta.db_table, from_columns, suffix=suffix
-                    )
-                ),
-                "column": ", ".join(
-                    [self.quote_name(from_col) for from_col in from_columns]
-                ),
-                "to_table": self.quote_name(field.target_field.model._meta.db_table),
-                "to_column": ", ".join(
-                    [self.quote_name(to_col) for to_col in to_columns]
-                ),
-                "deferrable": self.connection.ops.deferrable_sql(),
-            }
-        return super()._create_fk_sql(model, field, suffix)
-
-    # Override
-    def execute(self, sql, params=()):
-        # Hack: Citus will throw the following error if these statements are
-        # not executed separately: "ERROR: cannot execute multiple utility events"
-        if sql and not params:
-            for statement in str(sql).split(";"):
-                if statement and not statement.isspace():
-                    super().execute(statement)
-        elif sql:
-            super().execute(sql, params)
+class DatabaseSchemaEditor(
+    mixins.DatabaseSchemaEditorMixin, PostgresqlDatabaseSchemaEditor
+):
+    pass
```

Comment:
We should keep the base class for regular models. They are required for tenancy of regular models

Reply:
I don't understand this feedback. This base class is still here. The mixin is used to extract common functionality that is used by both Postgres and Postgis.